dash-spv-masternode-processor/src/models/llmq_entry.rs

```diff
@@ -315,7 +315,7 @@ impl LLMQEntry {
 
     fn validate_bitset(bitset: Vec<u8>, count: VarInt) -> bool {
         if bitset.len() != (count.0 as usize + 7) / 8 {
-            log_warn!(target: "masternode-processor",
+            log_debug!(target: "masternode-processor",
```

Comment:
MN processor writes a lot of logs, so I made them DEBUG for now. We can re-enable what is needed later, since no logs was written before anyway.